projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -6517,10 +6518,18 @@ export abstract class IgxGridBaseDirective extends DisplayDensityBase implements
             return;
         }
         if (commit) {
+            this.verticalScrollContainer.onDataChanged.pipe(first()).subscribe(() => {
+            // A check whether the row is in the current view
+            const shouldScroll = this.navigation.shouldPerformVerticalScroll(this.dataLength - 1, 0);
+            if (shouldScroll) {
+                this.addRowSnackbar.show();
```

Comment:
Instead of hiding the whole snackbar, maybe we can just show/hide the "SHOW" button inside. So that the grid will still show that row was added but will not give the option to scroll to the row as the row is already visible.